runbotics/runbotics-scheduler/src/scheduler-database/process-credential/process-credential.controller.ts

```diff
@@ -34,6 +35,17 @@ export class ProcessCredentialController {
         await this.processCredentialService.create(processCredentialDto, user);
     }
 
+    @Patch('processes/:processId')
+    update(
+        @Param('processId', ParseIntPipe) processId: IProcess['id'],
+        @Body(new ZodValidationPipe(editProcessCredentialArrayDto))
+        processCredentialsDto: EditProcessCredentialArrayDto,
+        @UserDecorator() user: User,
+    ) {
+        this.logger.log('REST request to update credentials order');
```

Comment:
We've decided not to log by hand -> incoming requests are being auto logged by Nest
check this out https://all41.atlassian.net/wiki/spaces/RPA/pages/2987196431/RB+Guidelines#Logi-typu-info